bluemira/equilibria/flux_surfaces.py

```diff
@@ -123,6 +123,51 @@ def connection_length(self, eq):
         """
         return np.sum(self._dl(eq))
 
+    def calculate_poloidal_angle(self, eq, x_strike, z_strike, gamma):
```

Comment:
`calculate` is not necessary in the function name.     

Let's remove it, it will be also have a better consistency with `connection_length` name.

```suggestion
    def poloidal_angle(self, eq, x_strike, z_strike, gamma):
```